x/blob/client/cli/payforblob.go

```diff
@@ -57,10 +59,20 @@ func CmdPayForBlob() *cobra.Command {
 	}
 
 	flags.AddTxFlagsToCmd(cmd)
+	cmd.PersistentFlags().Uint8(FlagNamespaceVersion, 0, "Specify the namespace version")
 
 	return cmd
 }
 
+func getNamespace(namespaceID []byte, namespaceVersion uint8) (appns.Namespace, error) {
+	switch uint8(namespaceVersion) {
+	case appns.NamespaceVersionZero:
+		return appns.New(uint8(namespaceVersion), append(appns.NamespaceVersionZeroPrefix, namespaceID...))
```

Comment:
this type cast is no longer necessary

```suggestion
		return appns.New(namespaceVersion, append(appns.NamespaceVersionZeroPrefix, namespaceID...))
```